src/openrct2/ride/RideRatings.cpp

```diff
@@ -2262,10 +2262,12 @@ static void RideRatingsApplyPenaltyLateralGs(RatingTuple& ratings, const Ride& r
 
 bool RatingTuple::isNull() const
 {
-    return excitement == kRideRatingUndefined;
+    return excitement == kRideRatingUndefined && intensity == kRideRatingUndefined && nausea == kRideRatingUndefined;
```

Comment:
No need for this, and it will break existing parks. Please revert.

Reply:
Ok, reverted.

Reply:
```suggestion
    return excitement == kRideRatingUndefined;
```